CONTRIBUTING.md

```diff
@@ -0,0 +1,5 @@
+# How to contribute
+
+Add whatever it is that you would like to the README.md. You can fork the repo and edit locally, or simply use Github's built-in editor. 
+
+Try to follow existing formatting. In a lot of cases, it makes sense to alphabetize the items. If a list is already in alphabetical order, please add new items in alphabetical order as well.
```

Comment:
I would concur with everything here especially on alphabetizing.  I think we will probably want to expand a like in line table of contents eventually.  Also, we should definitely keep an eye out for like sharks, but I think since we're using a pull-request format, we can probably avoid a lot of that.

As long as it's tech related or home grown valley companies that employ developers I really don't have any other concerns.  If they're a local company that outsources then I'm not into it.